views/result/account.validation.failure.html

```diff
@@ -14,4 +14,15 @@ <h3 class="govuk-notification-banner__heading">
     titleText: 'Failure',
     classes: 'govuk-notification-banner--failure',
     html: html
-}) }}
\ No newline at end of file
+}) }}
+
+{% block content %}
+    <h2 class="govuk-heading-m">What happens next</h2>
+    <p class="govuk-body">
+        You can <a href={{BASE_URL}}>validate another single file or zip file.</a>
+    </p>
+    <p class="govuk-body">
+        You can return to <a href={{CHS_URL}}>Companies House home page.</a>
+    </p>
+
+{% endblock %}
```

Comment:
We don't need the content block here as it is already in the [result view](https://github.com/companieshouse/account-validator-web/blob/729002e8d8687f6ce8b6d5ebe0d669a09896bf43/views/result/result.html#L18). The different validation results are then included in the result view with the [`showValidationResult()`](https://github.com/companieshouse/account-validator-web/blob/729002e8d8687f6ce8b6d5ebe0d669a09896bf43/views/result/result.html#L34).

```suggestion
    <h2 class="govuk-heading-m">What happens next</h2>
    <p class="govuk-body">
        You can <a href={{BASE_URL}}>validate another single file or zip file.</a>
    </p>
    <p class="govuk-body">
        You can return to <a href={{CHS_URL}}>Companies House home page.</a>
    </p>
```